src/components/exercise/group.cjsx

```diff
@@ -48,7 +55,7 @@ ExerciseGroup = React.createClass
     labels
 
   render: ->
-    {group, related_content, exercise_uid} = @props
+    {group, related_content, exercise_uid, project} = @props
```

Comment:
Should this project prop be defined as required in `propTypes` (but no default)?